middleware/index.ts

```diff
@@ -221,14 +217,16 @@ export default function applySfFxMiddleware(request: any, state: any, resultArgs
 
     //use secret here in lieu of DEBUG runtime environment var until we have deployment time support of config var
     const debugSecret = getSecret("sf-debug", "DEBUG", logger);
-    logger.info(`DEBUG flag is ${debugSecret}`);
     if(debugSecret || LoggerLevel.DEBUG === logger.getLevel() || process.env.DEBUG) {
+        logger.info(`DEBUG flag is ${debugSecret}`);
         //for dev preview, we log the ENTIRE raw request, may need to filter sensitive properties out later
         //the hard part of filtering is to know which property name to filter
         //change the logger level, so any subsequent user function's logger.debug would log as well
         logger.setLevel(LoggerLevel.DEBUG);
         logger.debug('debug raw request in middleware');
         logger.debug(request);
+    } else {
+        logger.info('DEBUG flag is off');
```

Comment:
If the log lines in the true block are omitted, then we know debug is not set.  I suppose I'm not sure why we need to state that debug is disabled.

Reply:
fixed per slack discussion - for support purposes it's better if we *always* log the DEBUG flag during dev preview - helps us know if we even hit that code section.